trunk/SUAVE/Methods/Weights/Correlations/UAV/__init__.py

```diff
@@ -1,2 +1 @@
-
```

Comment:
I renamed "Empty" to " empty" to match the other scrips in the Correlations folder however I forgot to change it in the _init_.py file. This issue has been rectified.